.github/workflows/verify_changes.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Get changed directories
         id: get-changeddir
         env:
-          TYPE: ${{ github.event_name }}
-          REF_NAME: ${{ github.ref_name }}
-          BASE: ${{ github.base_ref }}
+          TYPE: ${GITHUB_EVENT_NAME}
+          REF_NAME: ${GITHUB_REF_NAME}
+          BASE: ${GITHUB_BASE_REF}
         run: ./.github/scripts/verify_changes.sh ${{ env.TYPE }} ${{ env.REF_NAME }} ${{ env.BASE }}
```

Comment:
Regardless, the actual error seems here: I think we need to quote the environment variables.